modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/sql/DistributionZoneSqlDdlParserTest.java

```diff
@@ -48,48 +47,90 @@ public class DistributionZoneSqlDdlParserTest extends AbstractParserTest {
     @Test
     public void createZoneNoOptions() {
```

Comment:
let's derive all createZone test to `DistributionZoneOldSyntaxSqlDdlParserTest` to make sure nothing is broken